addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -423,11 +471,31 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
      * ie: The Job you apply for if the form is on that job's page.
      */
     addActionField: function (previewMode, value, params) {
```

Comment:
Maybe something to improve here for the following case: 
- Drop a form that "Create a Task" and add a field (+Field)
- Then set the project to one with properties
- On the field you just added, try to set it as a property field 
=> The property fields are not in the choices
- Add a new field
=> the choices are there for this one

Reply:
I added a selection of the form itself (to deselect the field), it seems to fix both this problem and the traceback from the other comment.